tools/cldr-apps/js/src/views/AddUser.vue

```diff
@@ -172,12 +173,21 @@ export default {
     },
 
     async validateLocales() {
+      if (!this.levelList) {
+        this.errors.push("Waiting for server...");
+        return;
+      }
+      const skipOrg = cldrUserLevels.canVoteInNonOrgLocales(
+        this.newUser.level,
+        this.levelList
+      );
+      const orgForValidation = skipOrg ? "" : this.newUser.org;
```

Comment:
The PR description says:
> [ for these 1 vote users ] do not send org to api/locales/normalize

That was very concerning to me if it sounded like normalization wasn't going to happen.  But I think this means that the org simply isn't going to be used as an input to the normalization.